### DISCUSSION TITLE
Implicit ID Token Flow in v6

### DISCUSSION THREAD

**Discussion by cgcoss** [panva/openid-client#739] [Q&A] [answered]
Hi, I have a question about migrating from v5 to v6.

After reading through the [breaking changes](https://github.com/panva/openid-client/releases/tag/v6.0.0) in v6, as well as comparing and contrasting the Examples, Quick Start and Certification sections of the [v5](https://github.com/panva/openid-client/tree/v5.7.1) and [v6](https://github.com/panva/openid-client/tree/v6.1.7) READMEs, I am under the impression that implicit id token flow is no longer supported. Is this correct?

I apologize if I have missed something obvious and thank you in advance for your help and support.

**Comment by panva** [comment, marked as answer]
> Is this correct?

Yes, it is.

**Reply by cgcoss** [reply]
Thank you for the quick response and for maintaining openid-client.
